proto/cmp/types/v1alpha/phone.proto

```diff
@@ -7,9 +7,12 @@ package cmp.types.v1alpha;
 // ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v1alpha/phone.proto.dot.xs.svg)
 // [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v1alpha/phone.proto.dot.svg)
 message Phone {
-  string phone = 1;
-  PhoneType phone_type = 2;
-  string phone_type_other = 3;
+  string number = 1;
+
+  PhoneType type = 2;
+
+  // Do we need this?
```

Comment:
Same here as for the e-mail - my guess is that it's used for `PHONE_TYPE_OTHER` or when you have multiple of the same type to have an additional field to describe it.